resources/js/common/components/PlayableListSortButton/PlayableListSortButton.tsx

```diff
@@ -68,6 +69,9 @@ export const PlayableListSortButton: FC<PlayableListSortButtonProps> = ({
 
     '-wonBy': { label: t('Won by (least)'), icon: LuUsers },
     wonBy: { label: t('Won by (most)'), icon: LuUsers },
+
+    '-rank': { label: t('Rank (lowest)'), icon: LuHash },
```

Comment:
@wescopeland - I was going to sugget using `LuArrowDownNarrowWide` and `LuArrowDownWideNarrow` for more consistency with other items in the menu, but then I realized the sort menu for achievements has items without arrows, so I'm not sure if it would be better. Do you have a preference?